sound/soc/sof/loader.c

```diff
@@ -176,6 +176,22 @@ static int ext_man_get_cc_info(struct snd_sof_dev *sdev,
 	return get_cc_info(sdev, &cc->cc_version.ext_hdr);
 }
 
+static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
+				    const struct sof_ext_man_elem_header *hdr)
+{
+	const struct ext_man_dbg_abi *dbg_abi;
+
+	dbg_abi = container_of(hdr, struct ext_man_dbg_abi, hdr);
+
+	dev_info(sdev->dev,
```

Comment:
isn't this a bit verbose, we are going to have one log for each syspend/resume cycle, no?
dev_dbg?

Reply:
updated

Reply:
@plbossart ok, I am sorry for raising this now instead of yesterday immediately after your comment, but what strikes me in this new version, is that now in the "normal" stock configuration, where `dev_dbg()` is disabled, this manifest entry will have 0 effect. Is this really what we want? As for every suspend / resume, it's now guarded by a `first_boot` test, so, maybe we do want `dev_info()` for this? Not something I feel strongly about, maybe that's indeed what we want, but just seems a bit weird to me.